lib/frameworks/utils.rb

```diff
@@ -95,5 +95,34 @@ def save_and_link_screenshot
       Capybara.current_session.driver.save_screenshot("./reports/Screenshot_#{current_time}.png")
       embed "./reports/Screenshot_#{current_time}.png", 'image/png', "Actual screenshot of the error at #{current_url}"
     end
+
+    def getting_console_errors(type)
```

Comment:
Please change the method name to get_console_logs

Reply:
agree.